### PR TITLE
chore(docs): link docs to Pocket codebase, YouTube

### PR DESCRIPTION
**Pull request by danieldreier**
Am I missing any YouTube videos you all are aware of?

### CODE REVIEW

**Review by skorfmann — APPROVED** [review]
There are likely more, but I think it's sufficient 👍

File: website/docs/cdktf/examples.html.md
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:81]:
```suggestion
[0.6: Snapshot testing](https://www.youtube.com/watch?v=9Is4QJT2664)
```

File: website/docs/cdktf/examples.html.md
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:83]:
```suggestion
If you're interested in extending the `cdktf convert` command, you may be interested in the [convert deep dive](https://www.youtube.com/watch?v=rSn4-Ki5nho). You do not need to watch this video to use the `convert` command successfully.
```

File: website/docs/cdktf/examples.html.md
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:87]:
```suggestion
Mozilla Pocket is a widely used application for managing reading lists that is built into the Firefox browser. Like many Mozilla projects, Pocket is open source, and the CDK for Terraform codebase that Pocket uses to manage infrastructure for the recommendation API is also public and open source. Pocket's codebase provides a great example of how to lay out a CDK for Terraform project.
```

File: website/docs/cdktf/examples.html.md
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:93]:
```suggestion
1. Read [`Pocket/recommendation-api/.aws/src/main.ts`](https://github.com/Pocket/recommendation-api/blob/main/.aws/src/main.ts).
2. Look at the constructs used that are defined in the [`Pocket/terraform-modules`](https://github.com/Pocket/terraform-modules/tree/main/src/pocket) repository, such as [`PocketALBApplication`](https://github.com/Pocket/terraform-modules/blob/main/src/pocket/PocketALBApplication.ts).
3. Look at the ["base" constructs](https://github.com/Pocket/terraform-modules/tree/main/src/base), which are are used in the higher-level constructs in the previous step.
```

File: website/docs/cdktf/examples.html.md
Comment (laurapacilio) [inline website/docs/cdktf/examples.html.md:89]: Did you mean from CDK for Terraform code? Just checking!

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block, but suggesting some nits for you to change. Also - I think we had discussed adding a few sentences of description for each of the examples earlier up on this page so that users know what they are clicking on. I imagine we could pull some context out of the READMEs in those examples. After this PR is merged, would it be helpful for me to pull that context and draft the 1-2 sentence explanations?

File: website/docs/cdktf/examples.html.md
Comment (danieldreier) [reply website/docs/cdktf/examples.html.md:89]: Yes! If you can draft example-summaries that would be a great improvement

**Pull request merged**
